yarn-project/foundation/src/testing/files/index.ts

```diff
@@ -5,16 +5,6 @@ import { createConsoleLogger } from '../../log/console.js';
 import { fileURLToPath } from '../../url/index.js';
 import { isGenerateTestDataEnabled } from '../test_data.js';
 
-let generateProtocolCircuitTestData = false;
-
-/**
- * This is separate so Prover.tomls don't get edited everytime any test is run,
- * Only full.test updates prover tomls, then switches this off.
- */
-export function switchGenerateProtocolCircuitTestData() {
-  generateProtocolCircuitTestData = !generateProtocolCircuitTestData;
-}
-
```

Comment:
Just catching up on some pre-Christmas stuff - doesn't the removal of this mean that we unecessarily generate huge `Prover.toml` files in most cases we run with `AZTEC_GENERATE_TEST_DATA`? I added this to prevent carrying around the huge files (they are pushed in `server.ts` everytime a rollup circuit is run)